src/modules/job-manager/plugins/perilog.c

```diff
@@ -248,7 +248,7 @@ static struct perilog_procdesc *perilog_procdesc_create (json_t *o,
     }
 
     pd->cmd = cmd;
-    pd->kill_timeout = kill_timeout > 0. ? kill_timeout : 5.;
+    pd->kill_timeout = kill_timeout > 0. ? kill_timeout : 10.;
```

Comment:
I think systemd `TimeoutStopSec` defaults to 90s, which for our systemd `flux-prolog@` and `flux-epilog@` services would be the time from SIGTERM induced `systemctl stop` to systemd SIGKILL.

Seems like we should coordinate this value to be greater than that.   If necessary, can reduce the default to whatever we want by adding `TimeoutStopSec` to the unit files.

Reply:
No argument that the timeout should be < 90s.

Are we ok with nodes getting drained after stuck prolog, then systemd SIGKILLing the cgroup 90s later?  We could reduce `TimeoutStopSec` in the unit files we ship if desired.

Reply:
That's a good question. I'm not sure of the right solution.

Reply:
Well maybe we can go with not changing anything in this release and see if that behavior is bothersome.

Reply:
Yeah, admins could always add an override file in a pinch.